frontend/simple/components/GroupMembers.vue

```diff
@@ -1,36 +1,109 @@
 <template>
-  <section>
-    <h3 class="title is-3"><i18n>Members</i18n></h3>
-    <ul>
-      <li v-for="(member, username) in profilesForGroup()">
-        {{ username }}
-      </li>
-    </ul>
-    <a class="button invite-button" @click.prevent="invite">
-      <span class="icon"><i class='fa fa-plus'></i></span>
-    </a>
+  <section class="gm-container">
+    <div class="level">
+      <h3 class="title"><i18n>Group Members</i18n></h3>
+    </div>
+    <div class="level">
+      <div class="level-left">
+        <ul class="columns is-mobile is-narrow is-multiline is-marginless">
+          <li v-for="(member, i) in profilesForGroupMock"
+            class="column is-narrow badge">
+            <h4 class="badge-name is-ellipsis is-size-7">
+              {{ member.username }}
+            </h4>
+
+            <img class="badge-img" :src="member.photo" alt="" />
+
+            <!-- TODO: How to handle color / multiple tags logic? -->
+            <span class="tag badge-tag is-ellipsis">
+              {{ member.pledge }}
+            </span>
+          </li>
+        </ul>
+        <div class="badge">
+          <a class="button action is-light" @click.prevent="invite">
+            <span class="icon"><i class="fa fa-plus has-text-success"></i></span>
+          </a>
+        </div>
+      </div>
+    </div>
   </section>
 </template>
 <style lang="scss" scoped>
-  h3.title {
+  $iconSize: 4.6rem;
+  $marginVertical: 0.3rem;
+
+  .gm-container {
+    margin: 4rem 0; // find better way to handle margins between dashboard elements
+  }
+
+  .title {
     margin-top: 30px;
   }
 
-  .invite-button {
-    border-radius: 999px;
-    background-color: #ECECEC;
-    color: #6CAD22;
-    height: 63px;
-    width: 63px;
+  .badge {
+    width: 6.25rem;
+    margin-right: 2rem;
+    margin-bottom: 2rem;
+    padding: 0;
+    text-align: center;
+
+    &-name {
+      margin-bottom: $marginVertical;
+    }
+
+    &-img {
+      display: inline-block;
+      width: $iconSize;
+      height: $iconSize;
+      border-radius: 50%;
+      margin-bottom: $marginVertical;
+    }
+
+    &-tag {
+      margin-top: $marginVertical;
+      width: 100%;
+    }
   }
-  .invite-button .icon i {
-    font-size: 36px;
+
+  .button.action {
+    height: $iconSize;
+    width: $iconSize;
+    border-radius: 50%;
+
+    .fa.fa-plus {
+      font-size: 2.5rem;
+      height: 2.25rem;
+    }
+
+    // How to I import missins from Bulma to be available here?
+    // Bulma mixins are in SASS but this is SCSS... ._.
```

Comment:
the syntax difference in itself shouldn't be a problem I think.
we could probably import the bulma mixins file in the components where it's used